CHANGELOG.md.~HEAD^~

```diff
@@ -0,0 +1,246 @@
+### 1.0.0 @ May 14 2015
```

Comment:
What's this file?

Reply:
Aah damn. Sorry it's junk. Removing.

Reply:
Done. Accidentally commited that file when rewriting the other PR (use `vc-revision-other-window` command in emacs to check out previous versions of a file)